config/config.toml

```diff
@@ -131,6 +131,7 @@ maxFailedCountForJob = 128
 warehouseServiceMaxRetryTimeinHr = 3
 
 [Warehouse]
+schemaVersion = "v1"
```

Comment:
Let's remove the default entry from here. Only if needed, we can drive from config. Otherwise we don't need to expose.